glsk/glsk-quality-check-ucte/src/test/java/com/powsybl/glsk/ucte/quality_check/GlskQualityProcessorTest.java

```diff
@@ -33,71 +32,71 @@ private InputStream getResourceAsInputStream(String resource) {
     void qualityCheckWithCorrectValue() {
         UcteGlskDocument ucteGlskDocument = UcteGlskDocument.importGlsk(getResourceAsInputStream(COUNTRYTEST));
         Network network = Network.read("testCase.xiidm", getClass().getResourceAsStream("/testCase.xiidm"));
-        ReporterModel reporter = new ReporterModel("defaultTask", "defaultName");
-        GlskQualityProcessor.process(ucteGlskDocument, network, Instant.parse("2016-07-28T23:30:00Z"), reporter);
+        ReportNode reportNode = ReportNode.newRootReportNode().withMessageTemplate("defaultTask", "defaultName").build();
+        GlskQualityProcessor.process(ucteGlskDocument, network, Instant.parse("2016-07-28T23:30:00Z"), reportNode);
 
-        assertTrue(reporter.getReports().isEmpty());
+        assertTrue(reportNode.getChildren().isEmpty());
     }
 
     @Test
     void qualityCheckWithError1() {
         UcteGlskDocument ucteGlskDocument = UcteGlskDocument.importGlsk(getResourceAsInputStream(FIRST_ERROR));
         Network network = Network.read("testCase.xiidm", getClass().getResourceAsStream("/testCase.xiidm"));
-        ReporterModel reporter = new ReporterModel("defaultTask", "defaultName");
+        ReportNode reporter = ReportNode.newRootReportNode().withMessageTemplate("defaultTask", "defaultName").build();
```

Comment:
There seems to be some `reporter` methods/variables still in the code, could you rename them also?

Reply:
renamed